client/my-sites/theme/main.jsx

```diff
@@ -326,14 +321,26 @@ const ThemeSheet = React.createClass( {
 		return <ThemeDownloadCard theme={ this.props.id } href={ this.props.download } />;
 	},
 
+	getDefaultOptionLabel() {
+		const { defaultOption, active: isActive, isLoggedIn, price } = this.props;
```

Comment:
Is there any reason for assigning value from `active `to`isActive` except the fact that `is` better meaning.
Maybe just update `active` name in `props`?